apps/docs/src/app/platform/component-docs/platform-forms/platform-input/platform-input-example/platform-input-auto-complete-validation-example.component.ts

```diff
@@ -29,31 +30,34 @@ export class PlatformInputAutoCompleteValidationExampleComponent implements OnIn
     @Input()
     open: boolean;
 
-    /**
-     * The template with which to display the individual listed items.
-     * Use it by passing an ng-template with implicit content. See examples for more info.
-     */
-    @Input()
-    itemTemplate: TemplateRef<any>;
+    @ViewChild('typeahead')
+    typeahead: PopoverComponent;
 
     constructor(private fb: FormBuilder) {}
```

Comment:
FormBuilder is unsued

Reply:
`submitted` and `state` properties are unused as well

Reply:
removed.